resin/models/tag.py

```diff
@@ -0,0 +1,308 @@
+import json
+
+from ..base_request import BaseRequest
+from .device import Device
+from ..settings import Settings
+from .. import exceptions
+
+
+class Tag(object):
+    """
+    This class is a wrapper for Tag models.
+
+    """
+
+    def __init__(self):
+        self.device = DeviceTag()
+        self.application = ApplicationTag()
+
+
+class BaseTag(object):
```

Comment:
Since we are going to be adding more tagged resources soon, would it make sense to split this into multiple files under a tag related folder?

Reply:
If we add more resources here, indeed we should split it but for now i think it will be better to keep it like this for consistency with others (like configvar or envvar)